internal/cloud/realm/client.go

```diff
@@ -30,6 +30,7 @@ type Client interface {
 	ImportDependencies(groupID, appID, uploadPath string) error
 	Diff(groupID, appID string, appData interface{}) ([]string, error)
 	DiffDependencies(groupID, appID, uploadPath string) (DependenciesDiff, error)
+	DependenciesInstallation(groupID, appID string) (DependenciesInstallation, error)
```

Comment:
what about `DependenciesImportStatus` or `DependenciesInstallStatus`? `DependenciesImportStatus` seems to make most sense to me considering this is only being used for `ImportDependencies` right now. @arahmanan

Reply:
my one caution here would be that we shouldn't name the client method based on how its used, we should name it for what it does.

I finally got up to speed took a look over on the `baas` side... I would vote one of the following:
* keep it as `DependenciesInstallation` ... I didn't realize that's how we referred to the `struct` in the backend, I can figure out how to reason what an installation means eventually
* use `DependenciesStatus` since that's what the API handler is called (`getDependenciesStatus`)

I think ultimately I like the latter, although then it introduces a stutter with the `Status` field (which could ultimately renamed to `State` or `Type` while keeping the `json` mapping the same `"status"`)

---

at the very least, I think this encourages us to remove the `type DependencyStatus string` as its overloading more of these words, and really doesn't add anything that a simple `string` couldn't do

Reply:
on a different note I guess simply naming it `DependenciesStatus` makes sense considering it's a request to the `dependencies/status` endpoint so seems most fitting for that.